sdk/webpubsub/Azure.Messaging.WebPubSub/src/swagger/WebPubSub.json

```diff
@@ -38,7 +38,7 @@
                     "webpubsub"
```

Comment:
Changing this file is terrifying.  Have we coordinated your changes with the known source of truth?  I'd strongly suggest swagger transforms instead if we haven't.

Reply:
Yes, we should transfer as much as possible. The problem is that some of these changes are language specific, e.g. hiding some members (so that handcrafted members can be added). I included @vicancy on the PR to take a look at the swagger changes.

Reply:
Any language specific changes should still be done with AutoRest.CSharp customizations first or README transforms otherwise so you don't need to reapply them when the swagger is updated for the next version, the swagger doesn't drift slightly between languages, etc.  I will tell you from my experience with Storage that you're building very painful technical debt here.